src/provider/controller/NotificationCenter.ts

```diff
@@ -33,6 +30,23 @@ export class WindowManager {
         fin.desktop.System.addEventListener('monitor-info-changed', this.onMonitorInfoChanged.bind(this));
     }
 
+
+    private setupTrayIcon() {
+        enum MouseButton {
+            LEFT = 0,
+            RIGHT = 2
+        }
+        type ClickEvent = {button: MouseButton};
+        const application = fin.Application.getCurrentSync();
+        const icon = 'https://openfin.co/favicon-32x32.png';
+        application.addListener('tray-icon-clicked', (event: ClickEvent) => {
```

Comment:
`@types/openfin` already defines an event type for this - `TrayIconClickedEvent`.